lecsum-be/app/core/llm_client.py

```
@@ -136,3 +153,14 @@ def build_quiz_multichain():
     get_retry_quiz_prompt(),
     QuizResponse, # 퀴즈 생성과 동일한 방식으로 재시험 생성
 )
+
+top_sentence_chain: Runnable = (
+    ChatPromptTemplate.from_template("""
```

Ruff (0.14.8)

`ChatPromptTemplate` may be undefined, or defined from star imports

(F405)